ngraph/core/include/ngraph/op/batch_to_space.hpp

```diff
@@ -27,8 +27,7 @@ namespace ngraph
             class NGRAPH_API BatchToSpace : public Op
```

Comment:
Can "ngraph/op/op.hpp" be includes instead of "ngraph/op/util/fused_op.hpp"?

Reply:
Good catch. Done